.fvm/flutter_sdk

```diff
@@ -0,0 +1 @@
+/Users/kamilsikora/fvm/versions/2.5.3
```

Comment:
you should add a file called `fvm_config.json` and inside 

`{
  "flutterSdkVersion": "2.8.0-3.1.pre",
  "flavors": {}
}`

Reply:
Also, we add `settings.json` and inside we define 

`{
    "dart.flutterSdkPath": ".fvm/flutter_sdk",
}`

Reply:
That will set fvm version for anyone that downloads this project. Also, having local paths ain't sleek

Reply:
![CleanShot 2021-12-06 at 22 31 51](https://user-images.githubusercontent.com/45424810/144925885-b6a25889-730b-432a-b03e-697179293437.png

There already is fvm_config.json but I'm using latest stable version of Flutter which is 2.5.3.
Should I use 2.8.0-3.1.pre instead?